src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ManifestTest.cs

```diff
@@ -964,5 +965,55 @@ class TestService : Service {{ public override Android.OS.IBinder OnBind (Androi
 			Assert.IsTrue (b.LastBuildOutput.ContainsText ($"AndroidManifest.xml(12,5): java{extension} error AMM0000:"), "Should recieve AMM0000 error");
 			Assert.IsTrue (b.LastBuildOutput.ContainsText ("Apps targeting Android 12 and higher are required to specify an explicit value for `android:exported`"), "Should recieve AMM0000 error");
 		}
+
+		// TODO: <uses-sdk android:minSdkVersion="32" android:targetSdkVersion="32" />
+		// Causes warning: D8 : warning : An API level of 32 is not supported by this compiler. Please use an API level of 31 or earlier
+		// Add a 32 parameter here when we get a newer version of r8.
+		[Test]
+		[TestCase (0, true)]
+		[TestCase (0, false)]
+		[TestCase (21, true)]
+		[TestCase (31, false)]
```

Comment:
Can we put 33 here now, or do we still get a build warning from r8? That previous comment was fairly old.

Reply:
Looks like this is working, I added a case for "AndroidDefaultTargetDotnetApiLevel" to always track the latest.